### ISSUE TITLE
Program doesn't seem to work since Nugs update

### ISSUE THREAD

**Issue by ak32423432423** [Sorrow446/Nugs-Downloader#29]
I've been trying to use this program since the new update on August 31st, but any URL I enter returns as an invalid URL. Any help is greatly appreciated and I love this tool you've created.

**Comment by dr01dy** [comment]
git clone https://github.com/Sorrow446/Nugs-Downloader.git
cd Nugs-Downloader
go build
adjust the config.json file
./main url

**Comment by ak32423432423** [comment]
> git clone https://github.com/Sorrow446/Nugs-Downloader.git cd Nugs-Downloader go build adjust the config.json file ./main url

Thanks a bunch, that fixed it!

**Comment by Sorrow446** [comment]
Fixed yesterday. I only update the binaries after a lot of commits, sorry. https://github.com/Sorrow446/Nugs-Downloader/commit/da9325de01298130f1f68b225332d0920f560664

**Comment by Lynnwood** [comment]
Also have this issue. Looking forward to the new binaries.

**Comment by davidgruhin** [comment]
Any fix for the new exclusive livestream page urls and the playback inside a Modal? https://play.nugs.net/livestream/33826/exclusive

**Comment by Sorrow446** [comment]
> Any fix for the new exclusive livestream page urls and the playback inside a Modal? https://play.nugs.net/livestream/33826/exclusive
> 
> I was able to extract using the yt-dlp, https://github.com/yt-dlp/yt-dlp after finding the master manifest for the HLS stream

Did the links used to be like `https://play.nugs.net/#/exclusive-livestreams/container/`?

**Comment by darcagn** [comment]
> > Any fix for the new exclusive livestream page urls and the playback inside a Modal? https://play.nugs.net/livestream/33826/exclusive
> > I was able to extract using the yt-dlp, https://github.com/yt-dlp/yt-dlp after finding the master manifest for the HLS stream
> 
> Did the links used to be like `https://play.nugs.net/#/exclusive-livestreams/container/...`?

That's correct, and if you insert the desired ID into that old URL format and feed that to Nugs-Downloader it still works.
